internal/pkg/fieldtags/testdata/src/tests/test.go

```diff
@@ -15,12 +15,11 @@
 package fieldtags
 
 type Person struct {
-	password, creds          string      `levee:"source"`               // want "tagged field: password, creds"
-	secret                   string      `json:"secret" levee:"source"` // want "tagged field: secret"
-	another                  interface{} "levee:\"source\""             // want "tagged field: another"
-	hasCustomFieldTag        string      `example:"sensitive"`          // want "tagged field: hasCustomFieldTag"
-	hasTagWithMultipleValues string      `levee:"value0,source,value2"` // want "tagged field: hasTagWithMultipleValues"
-	name                     string      `some_key:"non_secret"`
-	spaceAfterFinalQuote     string      `key:"value" `
+	password, creds          string `levee:"source"`               // want "tagged field: password, creds"
+	secret                   string `json:"secret" levee:"source"` // want "tagged field: secret"
+	hasCustomFieldTag        string `example:"sensitive"`          // want "tagged field: hasCustomFieldTag"
+	hasTagWithMultipleValues string `example:"sensitive,verbose"`  // want "tagged field: hasTagWithMultipleValues"
```

Comment:
This case changed because we do not support multiple `levee:` values.  We could, but it would need to be accounted for above.

Reply:
I think I'd prefer >2 values with the target value in the middle or at the end (to test that we're actually looping), but this is fine.